library/common/glfw/text_input_plugin.h

```diff
@@ -42,7 +42,7 @@ class TextInputPlugin : public KeyboardHookHandler {
   void SendStateUpdate(const TextInputModel &model);
 
   // Sends an action triggered by the Enter key to the Flutter engine.
-  void EnterPressed(const TextInputModel &model);
+  void EnterPressed(TextInputModel &model);
```

Comment:
Non-const reference arguments are forbidden by Google style:
http://go/cstyle#Reference_Arguments

You'll need to change it to a pointer.